### ISSUE TITLE
Template engine tests hitting known parallelism issue in GetVisualStudioInstances

### ISSUE THREAD

**Issue by marcpopMSFT** [dotnet/sdk#44878]
## Build Information
Build: https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_build/results?buildId=869444
Build error leg or test failing: Microsoft.DotNet.Cli.New.IntegrationTests.DotnetNewInstantiateTests.CanInstantiateTemplate_WithConditions_BasedOnFileName
Pull request: https://github.com/dotnet/sdk/pull/44859

## Error Message

```json
{
  "ErrorMessage": "57005",
  "ErrorPattern": "",
  "BuildRetry": false,
  "ExcludeConsoleLog": false
}
```

**Comment by BrennanConroy** [comment]
> Template engine tests hitting known parallelism issue in GetVisualStudioInstances

Is there an issue describing the problem? Are there people working on fixing it? We're hitting this in the aspnetcore repo a lot and as far as I'm aware we don't do anything with UWP which is how the sdk resolved the problem.

**Comment by tmat** [comment]
@baronfel @jaredpar should this issue be closed?

**Comment by baronfel** [comment]
I don't think so - the tracker is still making recent edits, which impliies it's still being hit.

**Comment by tmat** [comment]
So the fix above didn't fix it?
